main.tf

```diff
@@ -158,6 +158,7 @@ module "tls" {
 # Creates a public IP address and an Application Gateway for forwarding internet traffic to the GraphDB proxies/instances
```

Comment:
Have you tested deploy with API Gateway then switch to another one and vice versa ?